src/Kerbalism/UI/Monitor.cs

```diff
@@ -633,7 +631,7 @@ void Indicator_signal(Panel p, Vessel v, Vessel_info vi)
 			  String.Format("{0,-14}\t<b>{1}</b>\n", Localizer.Format("#KERBALISM_UI_DSNconnected"), conn.linked ?
 					Lib.Color("green", Localizer.Format("#KERBALISM_Generic_YES")) : Lib.Color("#ffaa00", Lib.Italic(Localizer.Format("#KERBALISM_Generic_NO")))),
 			  String.Format("{0,-14}\t<b>{1}</b>\n", Localizer.Format("#KERBALISM_UI_sciencerate"), Lib.HumanReadableDataRate(conn.rate)),
-			  String.Format("{0,-14}\t<b>{1}</b>\n", remotetech ? Localizer.Format("#KERBALISM_UI_delay") : Localizer.Format("#KERBALISM_UI_strength"), signal_str),
```

Comment:
If the RT delay display has been ditched then ignore this 😉 
otherwise this line should be added
``` remotetech ? String.Format("{0,-14}\t<b>{1}</b>\n", Localizer.Format("#KERBALISM_UI_delay"), delay_str) : "",```